### DISCUSSION TITLE
Reti IoT / reti separate

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#3298] [GENERALE - Discussioni generiche]
Ciao a tutti, sono in fase di modifiche network, volevo un vostro parere se era consigliabile o meno separate tutti i dispositivi Smarthome su una rete separata da quella standard di navigazione (magari su una rete IoT).
Non ho però ben chiaro però se comunicazioni con HA stesso, nel senso andrebbe anche lui posizionato nella rete IoT?

**Comment by snakuzzo** [comment]
Consiglio...lascia perdere, non serve praticamente a niente farsi ste seghe. Concettualmente non è completamente sbagliata come idea, ma devi saper metterci mano e devi anche avere un router che ti permetta di configurare il routing come si deve (se ti dovesse capitare di leggerlo in giro, lascia perdere chi dice di usare la rete guest, perché non ha capito una fava...non va bene), da quello che trovi in giro, sono decisamente di più le volte che la gente si è incasinata che le volte in cui è andato tutto liscio.

**Reply by Matty1184** [reply]
Concordo pienamente che reti Guess o altro non hanno senso.. la mia grande paura è poi di non riuscire a capire quali porte dialogano e fare le regole. Per quanto riguarda quei dispositivi che dialogano solo con un cloud, quindi non in locale, avrebbe senso spostarli lì o nemmeno?

**Reply by snakuzzo** [reply]
Guarda, proprio le porte non c'entrano assolutamente nulla con questo discorso :)
Come ti dicevo prima, è una questione di routing e non di porte (sono cose totalmente diverse).
Il discorso fatto vale per tutto

**Reply by Matty1184** [reply]
> Guarda, proprio le porte non c'entrano assolutamente nulla con questo discorso :)
> 
> Come ti dicevo prima, è una questione di routing e non di porte (sono cose totalmente diverse).
> 
> Il discorso fatto vale per tutto

Beh in genere se vuoi fare un ruoting fatto bene apri solo le porte necessarie a quella specifica comunicazione o servizio e non tutto!

**Reply by snakuzzo** [reply]
Ripeto...Il routing è una cosa...le porte sono un'altra

**Reply by bubez81** [reply]
Questa operazione si fa con le vlan e con router che permettono di gestirle, non è semplicissimo ma nemmeno impossibile.
Sta di fatto però che se vuoi "proteggerti" vuole dire che tutti i tuo i device domotici non devono comunicare con l'esterno.
Quindi può funzionare con quelli che riesci a gestire solo in locale...ma se hanno bisogno del cloud? 
Il concetto è interessante ma secondo me finisce che ti logori il cervello e non raggiungi lo scopo desiderato comunque.

**Reply by snakuzzo** [reply]
Esatto... è la stessa cosa che gli ho detto all'inizio

**Reply by bubez81** [reply]
Si infatti, ho solo fatto un paio di aggiunte per chiarire meglio il concetto
